src/frontend/pages/game.js

```diff
@@ -34,14 +43,16 @@ export default function Home() {
         <div className={styles.actionBar}>
           <div className={styles.actionBarLeft}>
             {/* TODO: both of these values will be given by an API/socket */}
-            <h4>You Are: </h4>
-            <h4>Current Coins: </h4>
+            {/* TODO: add some JS to change styling of the color based on what color it is  */}
+            <h4 className={styles.actionBarText}>
+              You Are: <span style={{ color: 'green' }}>{thisColor} </span>
+            </h4>
+            <h4 className={styles.actionBarText}>
+              Current Coins:{' '}
+              <span style={{ color: '#546ec9' }}>{theseCoins} </span>
```

Comment:
Could we also make this color a constant

Reply:
+1 Let's make a constants file with all our colors and stuff

Reply:
I would also be in support of making like a variables.css file for some constants for css files